wire-grpc-client/src/main/java/com/squareup/wire/internal/RealGrpcCall.kt

```diff
@@ -62,19 +61,13 @@ internal class RealGrpcCall<S : Any, R : Any>(
           var message: R? = null
           var exception: IOException? = null
           try {
-            response.use {
-              response.messageSource(grpcMethod.responseAdapter).use { reader ->
-                message = reader.read()
-                    ?: throw ProtocolException("expected one message in response")
-                exception = response.grpcStatusToException()
```

Comment:
What happens to the trailers? we still have to check, don't we?

Reply:
We do! I deleted too much! Can fix.